test/hotspot/jtreg/compiler/vectorization/TestFloat16VectorOps.java

```diff
@@ -80,6 +82,8 @@ public void checkResultAdd() {
 
     @Test
     @Warmup(10000)
+    @IR(counts = {IRNode.SUB_VHF, ">= 1"},
+        applyIfCPUFeature = {"avx512_fp16", "true"})
```

Comment:
This rule can be merged with the next rule for sve - 
`applyIfCPUFeatureOr = {"avx512_fp16", "true", "sve", "true"}`. Same for other operations below.